crates/dojo-lang/src/inline_macros/utils.rs

```diff
@@ -0,0 +1,33 @@
+use std::collections::HashMap;
+use std::sync::Mutex;
+
+use cairo_lang_syntax::node::ast::{ExprPath, ExprStructCtorCall};
+use cairo_lang_syntax::node::kind::SyntaxKind;
+use cairo_lang_syntax::node::SyntaxNode;
+
+type ModuleName = String;
+type FunctionName = String;
+lazy_static::lazy_static! {
+    pub static ref SYSTEM_WRITES: Mutex<HashMap<ModuleName, HashMap<FunctionName, Vec<SystemRWOpRecord>>>> = Default::default();
+    pub static ref SYSTEM_READS: Mutex<HashMap<ModuleName, Vec<String>>> = Default::default();
+}
+
+pub enum SystemRWOpRecord {
+    StructCtor(ExprStructCtorCall),
+    Path(ExprPath),
+}
+
+pub fn get_parent_of_kind(
```

Comment:
```suggestion
pub fn parent_of_kind(
```

idomatic rust avoids the `get_` prefix